### PR TITLE
Modify integration tests to be compatible with 4.0.0

### PR DESCRIPTION
**Pull request by wasuradananjith**
## Purpose
The main purpose is to add the step/functionality to create and deploy API revisions when running the tests. Apart from that, minor changes have been done to match with APIM 4.0.0 use cases.

### CODE REVIEW

File: import-export-cli/integration/envSpecific_test.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/wso2/product-apim-tooling/import-export-cli/integration/testutils"
 )
 
-func TestEnvironmentSpecificParamsEndpoint(t *testing.T) {
+func TestXyz(t *testing.T) {
```
Comment (Chamindu36) [inline import-export-cli/integration/envSpecific_test.go:32]: Shall we add a proper test name here?

**Review by Chamindu36 — APPROVED** [review]

File: import-export-cli/integration/envSpecific_test.go (hunk 1 above)
Comment (wasuradananjith) [reply import-export-cli/integration/envSpecific_test.go]: Thanks. Fixed via 1f2616e92cf314fb83d78c87ab8df456ed066a28

File: import-export-cli/integration/apim/apim.go [hunk 2]
```diff
@@ -608,6 +608,63 @@ func (instance *Client) AddAPIProductFromJSON(t *testing.T, path string, usernam
 	return apiProductResponse.ID
 }
 
+// CreateAPIRevision : Create API revision
+func (instance *Client) CreateAPIRevision(apiID string) *APIRevision {
+	apiProductsURL := instance.publisherRestURL + "/apis/" + apiID + "/revisions"
```
Comment (uvindra) [inline import-export-cli/integration/apim/apim.go:613]: Lets change `apiProductsURL` to something generic like `revisioningURL`

File: import-export-cli/integration/apim/apim.go [hunk 3]
```diff
@@ -608,6 +608,63 @@ func (instance *Client) AddAPIProductFromJSON(t *testing.T, path string, usernam
...
+
+	var revision APIRevision
+	json.NewDecoder(response.Body).Decode(&revision)
+
+	return &revision
+}
+
+// DeployAPIRevision : Deploy API revision
+func (instance *Client) DeployAPIRevision(t *testing.T, apiID string, revision *APIRevision) {
+	apisURL := instance.publisherRestURL + "/apis/" + apiID + "/deploy-revision"
```
Comment (uvindra) [inline import-export-cli/integration/apim/apim.go:635]: Lets change `apisURL` to `deployURL`

File: import-export-cli/integration/apim/apim.go (hunk 2 above)
Comment (uvindra) [reply import-export-cli/integration/apim/apim.go:613]: I think I have made the same mistake in CreateAPIProductRevision(), please fix it for me :)

File: import-export-cli/integration/apim/apim.go (hunk 3 above)
Comment (uvindra) [reply import-export-cli/integration/apim/apim.go:635]: I think I have made the same mistake in DeployAPIProductRevision(), please fix it for me :)

File: import-export-cli/integration/envSpecific_test.go (hunk 1 above)
Comment (uvindra) [inline import-export-cli/integration/envSpecific_test.go]: Why is this called TestXyz?

Comment (wasuradananjith) [reply import-export-cli/integration/envSpecific_test.go]: This is already fixed via https://github.com/wso2/product-apim-tooling/pull/639/commits/1f2616e92cf314fb83d78c87ab8df456ed066a28

File: import-export-cli/integration/apim/apim.go (hunk 2 above)
Comment (wasuradananjith) [reply import-export-cli/integration/apim/apim.go]: Sure. Fixed via https://github.com/wso2/product-apim-tooling/pull/639/commits/9575af27ae7dcd8620fac2dba417df0c30eec04a

**Review by uvindra — APPROVED** [review]

**Pull request merged**